docs/guide/installation-procedure.md

```diff
@@ -304,6 +304,17 @@ open ~/Applications/Webots.app    # to launch Webots using the open command
 
 Alternatively, you can double-click on the Webots icon to launch it.
 
+#### Trouble Shooting for Mac M1 Users
```

Comment:
I wouldn't mention M1 only as M2 processors are likely to show up the same problem.

```suggestion
#### Troubleshooting for Apple Silicon Users
```

Reply:
Sounds good. Thank you.